.gitignore

```diff
@@ -2,9 +2,10 @@
 *.pyo
 *.pyc
 __pycache__
-/artifacts
```

Comment:
Not sure why this gets removed.
An own directory gets created for each nevr only during integration tests so far.  When running `ansible-playbook ...` command the results get saved to `artifacts`.

Reply:
Are you overriding `artifacts` variable on the command line? 
`ansible-playbook ... -e artifacts=path`

Reply:
Nevermind, maybe I misunderstood the question. Either way, I also believe it's better to keep this in .gitignore. When somebody runs the playbook with default vars, that's where the artifacts get created.

Reply:
I am not overriding `artifacts`, I just took README and tried running as it says :)
I will add it back.

Besides, while `test.log` gets overridden with each subsequent run, `output.log` keeps previous results (also with default vars).

Reply:
ok, get it back, my bad, I haven't considered direct runs.

Reply:
@irushchyshyn When running it locally, it's your duty to clear artifacts dir (or not). The task can't know your intended use case. Of course, you can e.g. generate random artifacts dir paths, in case the variable is not set externally. It's your task, you can do whatever you prefer for local runs :) Local runs are mostly for development.

Reply:
@kparal thanks for explanation.